lib/utilities.bash

```diff
@@ -60,15 +60,21 @@ function _bash-it-array-dedup() {
 	printf '%s\n' "$@" | sort -u
 }
 
-# Outputs a full path of the grep found on the filesystem
+# Runs `grep` with *just* the provided arguments
 function _bash-it-grep() {
-	: "${BASH_IT_GREP:=$(type -P egrep || type -P grep)}"
-	printf "%s" "${BASH_IT_GREP:-/usr/bin/grep}"
+	: "${BASH_IT_GREP:=$(type -P grep)}"
+	"${BASH_IT_GREP:-/usr/bin/grep}" "$@"
 }
 
-# Runs `grep` with extended regular expressions
+# Runs `grep` with fixed-string expressions (-F)
+function _bash-it-fgrep() {
+	: "${BASH_IT_GREP:=$(type -P grep)}"
+	"${BASH_IT_GREP:-/usr/bin/grep}" -F "$@"
+}
+
+# Runs `grep` with extended regular expressions (-E)
 function _bash-it-egrep() {
-	: "${BASH_IT_GREP:=$(type -P egrep || type -P grep)}"
+	: "${BASH_IT_GREP:=$(type -P grep)}"
 	"${BASH_IT_GREP:-/usr/bin/grep}" -E "$@"
 }
```

Comment:
Removed the `type -P egrep` usage.
Its weird to me that, under everyday conditions, this code would up invoking `egrep -E` ?